Update README (#46)
flexport/latitude: README.md

Latitude is an open source design system built by [Flexport][0].

> > Latitude is currently designed only for internal use at Flexport. If you're not

Reviewer: This formatting is a nice touch

Author: 💅

at Flexport, feel free to poke around, but we don't recommend using Latitude.
(At least, not yet.)

## 🚀 Overview
Our mission at Flexport is to make trade easy for everyone. For those of us charged with building the technologies to realize that mission, this means creating intuitive, scalable, and reliable experiences that users love. The foundation for those experiences is our design system: Latitude.

> Latitude was developed to give us a standard set of design principles and shared design elements. It enables us with the flexibility to design for a diverse range of products and features that will delight our users. It provides us a common visual language that ensures quality and consistency – aligned with our brand, across our organization. What’s more, Latitude helps us work efficiently, without having to continually recreate assets or patterns.

Reviewer: nit: This bit sounds very wordy; here are some minor suggestions. Feel very free to not take them, though!

```suggestion
Latitude was created to give Flexport a standard set of design principles and elements. It provides flexibility to design a diverse range of user-delighting products. It is a common visual language that ensures quality and consistency – aligned with our brand, across our organization. What’s more, Latitude helps us work efficiently, without having to continually recreate assets or patterns.
```

We’ve built Latitude to drive great user experiences with the flexibility to foster innovation. Learn more from our [introductory blog post][1].

## 🙌 Contributing
Latitude is currently designed only for **internal use at Flexport**. If you're not
at Flexport, feel free to poke around, but we don't recommend using Latitude.
(At least, not yet.)

📣 Flexport engineers: we believe that our design system is strongest when everyone
contributes. Check out the [contributing doc][2] for a getting started guide.

## 📚 Documentation

> - Our [documentation site][3] is the source of truth for our design guidelines and react components. 

Reviewer: ```suggestion
- Our [documentation site][3] is the source of truth for our design guidelines and React components. 
```

- The site is built from our monorepo (for now) but component docs are generated from the sourcecode in this repo!

## 📃 License
Latitude is [MIT licensed][4].

[0]: https://www.flexport.com

> [1]: https://medium.com/flexport-design/introducing-latitude-ab04c6e94c0c

Reviewer: Would be cool to publish this on https://flexport.engineering for better branding

Reviewer: Even better, we could do another post and link it to this project. 

The post doesn't have to be "here's our design system, please use it", but instead "here's how we approached building a design system for internal use". Since the project is now in the open, we can now easily link to code etc. Perhaps this is a more engineering-focussed post. @FermiDirak what do you think? (just an idea, no pressure)
